Cargo.toml

```diff
@@ -32,6 +32,7 @@ mopa = "0.2"
 shred = "0.4"
 shred-derive = "0.3"
 tuple_utils = "0.2"
+rayon = { version = "0.7" }
```

Comment:
Given that we don't use any unstable features anymore, let's not use an inline table but just a string.